### ISSUE TITLE
Please rename the repo name to `glow.yazi`

### ISSUE THREAD

**Issue by sxyazi** [Reledia/glow.yazi#1]
It would be great if the repo name could be renamed to `glow.yazi` and place `init.lua` at the root. This way users can easily install it with just a `git clone`, without the need for additional file moving operations.

Please also add a `README.md`, explaining how to use it, something like this:

```toml
[plugin]
prepend_previewers = [
  { name = "*.md", exec = "glow" },
]
```

**Comment by Reledia** [comment]
Added README and changed directory. Please let me know if it's ok now

**Comment by sxyazi** [comment]
Thank you! I've made some other changes, https://github.com/Reledia/glow.yazi/pull/2

**Comment by Reledia** [comment]
Merged in #2. Thanks for the help!
